app/gateway/resource-sizing-guidelines.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "{{site.base_gateway}} resource sizing guidelines"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+min_version:
+    gateway: 3.4.x
+
+tags:
+    - performance
+
+breadcrumbs:
+    - /gateway
+
+description: "See Kong's recommended resource allocation sizing guidelines for {{site.base_gateway}} based on configuration and traffic patterns."
```

Comment:
```suggestion
description: "Review Kong's recommended resource allocation sizing guidelines for {{site.base_gateway}} based on configuration and traffic patterns."
```
I like "review" here more, "see" sounds like you're about to link/direct them to another page.